src/OrbitGl/TimeGraph.h

```diff
@@ -128,8 +128,7 @@ class TimeGraph : public orbit_gl::CaptureViewElement, public orbit_gl::Timeline
   [[nodiscard]] const TimeGraphLayout& GetLayout() const { return layout_; }
   [[nodiscard]] TimeGraphLayout& GetLayout() { return layout_; }
 
-  // TODO(http://b/194777907): Move GetColor outside TimeGraph
-  [[nodiscard]] static Color GetColor(uint32_t id) {
+  [[nodiscard]] static Color GetColor(uint64_t id) {
     constexpr unsigned char kAlpha = 255;
```

Comment:
I'm a bit unsure about if we should use the new method to not have duplicate implementation. @beckerhe what do you think?

Reply:
I would say it depends on why the implementation is duplicate. If the implementation just happened to be the same and if it was fine to change one of them in the future independently from the other, then I would leave it as a duplicate.

But if both functions need to return the same color for the same ID and changing only one of them was breaking the code, then I would use the new function here as well.

Does that make sense?

Reply:
I think I'll leave it as it is right now, since we can have two different independent implementations for thread colors and non thread colors.